day_two_guide/topics/controller_role_verification.adoc

```diff
@@ -12,31 +12,39 @@ hosts. The service runs in active/passive mode, meaning it should only be
 running on one master at any time.
 
 The {product-title} controllers execute a procedure to choose which host runs
-the service. The current running value is stored in the etcd cluster within the `/openshift.io/leases/controllers` key.
+the service. The current running value is stored in an annotation in a special
+`configmap` stored in the `kube-system` project.
 
 Verification of the master host actively running the
 `atomic-openshift-master-controllers.service` service is performed by running
-the following command on any of the master nodes:
+the following command as `cluster-admin`:
```

Comment:
@e-minguez Can I suggest:

Verify the master host by running the `atomic-openshift-master-controllers.service` service as a `cluster-admin`:

Reply:
Done!